app/controllers/application_controller.rb

```diff
@@ -82,6 +82,17 @@ def request_event
     @request_event ||= RequestEvent.new(request, response, session, current_jobseeker, current_publisher_oid)
   end
 
+  def trigger_feedback_provided_event
+    feedback_data = feedback_params.to_h.map.each { |key, value|
```

Comment:
`feedback_params` should be already a hash, no need to do `to_h.map` you should be able to just do `feedback_params.each`

Reply:
```ruby
undefined method `map' for #<ActionController::Parameters:0x00007fb8e9a4d680>
```